src/components/Highlights/index.tsx

```diff
@@ -52,27 +52,7 @@ export function Cards({ from: cards }: { from: CardProps[] }) {
 	return (
 		<section className="margin-top--lg margin-bottom--xl">
 			<div className="container">
-				<div className={clsx("margin-bottom--md")}>
-					{ cards.length ? <Heading as="h2">Our favorites ✨</Heading> : <></> }
```

Comment:
Sorry I just don't think this make sense lol. Will you come around or do you really really want this in? I think having two headings is kind of redundant, maybe something like this would make more sense on the community plugins showcase down the road?

Reply:
I don't oppose removing the title, just feel that it would look better if there was some padding between the card and the header. The title (text) is the method I can think of to fill that spacing, but maybe there are other things, or should we simply go with no spacing for now?

Reply:
I think we can go with no spacing for now and merge this. I'll take a look later and try some things out but let's just finish this PR off 😅